Makefile

```diff
@@ -243,7 +243,7 @@ coverage-clean: ## Remove coverage artifacts.
 	$(RM) -r coverage/tmp
 	@if [ -d "out/Release/obj.target" ]; then \
 		$(FIND) out/$(BUILDTYPE)/obj.target \( -name "*.gcda" -o -name "*.gcno" \) \
-			-type f -exec $(RM) {};\
+			-type f -exec $(RM) {} \; ; \
```

Comment:
```suggestion
			-type f | xargs $(RM); \
```

Reply:
Thank you for the review. It would be better as you reviewed.